file-formats/uist/examples/z_aff_example_uist.uist.json

```diff
@@ -7,7 +7,7 @@
   "generalInformation": {
     "title": "Title Example of UIST Object for ABAP File Format",
     "sortPriority": 1.001,
-    "baseSpaceTemplateName": "BASE_SPACE"
+    "mergeId": "Z_MERGE_1"
```

Comment:
Maybe I was too fast with my approval. Is ID referring to another object name. I guess it is a reference to another space template, isn't it? 

Usually, other file formats use the term "...Name" for object names.

Reply:
Thanks for clarification. I guess I got confused, because it replaced a space template name and the Id in the example looked like an object name.